package.json

```diff
@@ -11,7 +11,8 @@
     "start": "node server.js",
     "dev": "concurrently \"npm run start\" \"grunt clientDev\"",
     "postinstall": "bower install && grunt clientBuild",
-    "test": "grunt test"
+    "test": "grunt test",
+    "lint": "./node_modules/eslint/bin/eslint.js web/js/"
```

Comment:
We could just make this:
`"lint": "./node_modules/eslint/bin/eslint.js ."`
and then it will lint the server side and client js in one go.
The linter will correctly select the relevant `.eslintrc` config depending on the file location without any extra commands.